presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -86,11 +92,43 @@ fun EventScreen(
                     )
 
                     Spacer(modifier = modifier.height(26.dp))
+                    AgendaDropDownMenu(
+                        modifier = Modifier
+                            .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+                        shouldOpenDropdown = eventScreenState.shouldOpenDropdown,
+                        onCloseDropdown = {
+                            eventScreenEvent(
+                                EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
+                        },
+                        listOfMenuItemId = listOf(
+                            R.string.ten_minutes_before,
+                            R.string.thirty_minutes_before,
+                            R.string.one_hour_before,
+                            R.string.six_hours_before,
+                            R.string.one_day_before),
```

Comment:
You already created that nice enum class, why not use it here?
```
AlarmReminderItem.values().map { it.text }
```
Otherwise, updating your enum won't change what's shown in this drop down

Reply:
@philipplackner Updating to use the map.